tests/pyoozie/test_tags.py

```diff
@@ -148,28 +147,28 @@ def _xml(self, doc, tag, text):
 
 
 def test_parameters(expected_property_values, expected_property_values_xml):
-    actual = Parameters(expected_property_values).xml(indent=True)
+    actual = tags.Parameters(expected_property_values).xml(indent=True)
     expected = '<parameters>{xml}</parameters>'.format(xml=expected_property_values_xml)
-    assert xml_to_dict_unordered(expected) == xml_to_dict_unordered(actual)
+    assert tests.utils.xml_to_dict_unordered(expected) == tests.utils.xml_to_dict_unordered(actual)
```

Comment:
I wanted to avoid importing `utils`; it doesn't communicate what sort of utils that we're using. The term is too generic. I would like to emphasize that these are testing utilities as opposed to the many other utilities that we may one day import.